src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -139,9 +147,10 @@ function updateBundleExcessSlowFills(
 }
 
 function updateBundleSlowFills(dict: BundleSlowFills, deposit: V3DepositWithBlock & { lpFeePct: BigNumber }): void {
-  if (chainIsEvm(deposit.destinationChainId) && !isValidEvmAddress(deposit.recipient)) {
-    return;
-  }
+  assert(
+    chainIsEvm(deposit.destinationChainId) && isValidEvmAddress(deposit.recipient),
```

Comment:
If you are adding this assert, I believe you will also need to change `_canCreateSlowFillLeaf` to check for a valid recipient since somebody can supply garbage to a deposit and request a slow fill for that.

Reply:
This is a good point I'm going to revert 5c32bd80eafa3faf05788d2718a09766dddbc7a9